x/concentrated-liquidity/pool.go

```diff
@@ -519,3 +519,80 @@ func (k Keeper) MigrateAccumulatorToScalingFactor(ctx sdk.Context, poolId uint64
 	ctx.Logger().Info("migration end", "pool_id", poolId)
 	return nil
 }
+
+// MigrateSpreadFactorAccumulatorToScalingFactor multiplies the value of the spread reward accumulator, respective position accumulators
+// and tick uptime trackers by the per-unit liquidity scaling factor and overwrites the accumulators with the new values.
+func (k Keeper) MigrateSpreadFactorAccumulatorToScalingFactor(ctx sdk.Context, poolId uint64) error {
```

Comment:
This method is pretty key logic, but generally follows the same flow as migrating incentive accumulators.